discord/app.js

```diff
@@ -22,9 +22,22 @@ const client = new Client({
 });
 
 // n1 Confirm client is ready in terminal
-client.once(Events.ClientReady, readyClient => {
-    console.log(`Ready! Logged in as ${readyClient.user.tag}`);
-});
+module.exports = {
+    listenToMessages: (onMessage) => {
+        client.once(Events.ClientReady, readyClient => {
+            console.log(`Ready! Logged in as ${readyClient.user.tag}`);
+        });
```

Comment:
This function should be outside of `module.export`, as it fires when the bot first starts up.

Maybe cut & paste this above `module.export`?